Dockerfile

```
@@ -0,0 +1,25 @@
+FROM python:alpine
```

Please always make sure to use a well defined base image.
`python:alpine` is currently based on python3.7, what is going to happen in the future?
If you check https://github.com/macisamuele/gdgpisausermanager/blob/29e2ce660c30f0e4c94fcf2dfc2665ade6e903b0/Dockerfile you can see that I was defining a specific python and alpine version.
Build reproducibility is important (at least to me 😉 )

---

Thank you for the suggestion!
Do you think that this can fix?
```suggestion
FROM python:3.7-alpine
```

---

It could work as well (`python:3.7.3-alpine3.8` would be even better)

---

Perfect! Thanks. I'll upgrade to alpine 3.9 since is the latest and works correctly on that.